boa_tester/src/exec/mod.rs

```diff
@@ -67,6 +74,7 @@ impl TestSuite {
             passed += suite.passed;
             ignored += suite.ignored;
             panic += suite.panic;
+            features.append(&mut suite.features.clone())
```

Comment:
This is giving a clippy warning, should be fixed with this:

```suggestion
            features.append(&mut suite.features.clone());
```

Reply:
Thanks. Applied https://github.com/boa-dev/boa/pull/1980/files#diff-fc3d7ad7b5e64574258c9febbe56171f3309b74e0c8da35238a76002f3ee34d9R77